### DISCUSSION TITLE
grid with datetime column with value null

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#7270] [Help]
Hi,
I have a grid with a datetime column. sometimes this column has a null value and is represented in the grid as “01/01/1901”.  
![image](https://github.com/user-attachments/assets/3a6ae22c-9263-40f5-91db-44e06762071d)
Is there any way to make it not show anything in these situations?
Thank You

**Comment by luongnv6** [comment]
grid only shows data fetched from DB, if it is null it will not show anything. Please check your data or show code so everyone can check
